recipe/meta.yaml

```
@@ -1,32 +1,32 @@
-{% set tbb_version = "2021.13.0" %}
+{% set tbb_version = "2021.13.1" %}
 # this variable defines oneMKL specific version
-{% set mkl_version = "2024.2.0" %}
+{% set mkl_version = "2024.2.1" %}
 # this variable defines oneMKL specific buildnum
-{% set mkl_buildnum = "663" %}  # [linux]
-{% set mkl_buildnum = "661" %}  # [win]
+{% set mkl_buildnum = "105" %}  # [linux]
```

I bet it must be 103:

```suggestion
{% set mkl_buildnum = "103" %}  # [linux]
```

```bash
conda search -c https://software.repos.intel.com/python/conda --override 'mkl=2024.2.1' --info
Loading channels: done
mkl 2024.2.1 intel_103
----------------------
file name   : mkl-2024.2.1-intel_103.tar.bz2
name        : mkl
version     : 2024.2.1
build       : intel_103
build number: 103
size        : 151.5 MB
license     : Intel Simplified Software License
subdir      : linux-64
url         : https://software.repos.intel.com/python/conda/linux-64/mkl-2024.2.1-intel_103.tar.bz2
md5         : 04ce5f48b44ebb9f9b3990f5cc68797d
timestamp   : 2024-08-01 01:19:09 UTC
dependencies:
  - intel-openmp >=2023*
  - tbb 2021.*
```

If it interferes with other package that uses same variable, please split this variable in two different ones.